tools/annotation_tools/turk_with_s3/get_results.py

```diff
@@ -45,67 +36,97 @@ def get_hit_list_status(mturk):
-                        answer = answer_field["FreeText"]
+                    print("Worker's answer was:")
+                    if type(xml_doc["QuestionFormAnswers"]["Answer"]) is list:
+                        # Multiple fields in HIT layout
+                        for answer_field in xml_doc["QuestionFormAnswers"]["Answer"]:
+                            input_field = answer_field["QuestionIdentifier"]
+                            answer = answer_field["FreeText"]
+                            print("For input field: " + input_field)
+                            print("Submitted answer: " + answer)
+                            new_row["Answer.{}".format(input_field)] = answer
+
+                        res = res.append(new_row, ignore_index=True)
+                        res.to_csv(output_csv, index=False)
+                    else:
+                        # One field found in HIT layout
+                        answer = xml_doc["QuestionFormAnswers"]["Answer"]["FreeText"]
+                        input_field = xml_doc["QuestionFormAnswers"]["Answer"]["QuestionIdentifier"]
                         print("For input field: " + input_field)
                         print("Submitted answer: " + answer)
                         new_row["Answer.{}".format(input_field)] = answer
+                        res = res.append(new_row, ignore_index=True)
+                        res.to_csv(output_csv, index=False)
 
-                    res = res.append(new_row, ignore_index=True)
-                else:
-                    # One field found in HIT layout
-                    answer = xml_doc["QuestionFormAnswers"]["Answer"]["FreeText"]
-                    input_field = xml_doc["QuestionFormAnswers"]["Answer"]["QuestionIdentifier"]
-                    print("For input field: " + input_field)
-                    print("Submitted answer: " + answer)
-                    new_row["Answer.{}".format(input_field)] = answer
-                    res = res.append(new_row, ignore_index=True)
-
-                mturk.approve_assignment(AssignmentId=assignment["AssignmentId"])
+                    mturk.approve_assignment(AssignmentId=assignment["AssignmentId"])
+                    mturk.delete_hit(HITId=hit_id)
+            else:
+                print("No results ready yet")
+                # if returned assignment is empty,reject
                 mturk.delete_hit(HITId=hit_id)
-        else:
-            print("No results ready yet")
-    curr_hit_status = get_hit_list_status(mturk)
-    print(curr_hit_status)
+        curr_hit_status = get_hit_list_status(mturk)
+        print(curr_hit_status)
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--output_csv", type=str, required=True)
+    parser.add_argument("--dev", action="store_true")
+
+    args = parser.parse_args()
+    if args.dev:
+        MTURK_SANDBOX = "https://mturk-requester-sandbox.us-east-1.amazonaws.com"
```

Comment:
nit: `MTURK_SANDBOX` -> `MTURK_URL` to name it better since the other one isn't a sandbox